funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_worker.py

```diff
@@ -90,6 +91,11 @@ def __init__(self, worker_id, address, port, logdir, debug=False, worker_type='R
         logger.info('Trying to connect to : tcp://{}:{}'.format(self.address, self.port))
         self.task_socket.connect('tcp://{}:{}'.format(self.address, self.port))
         self.poller.register(self.task_socket, zmq.POLLIN)
+        signal.signal(signal.SIGTERM, self.handler)
+
+    def handler(self, signum, frame):
+        logger.error("Signal handler called with signal", signum)
+        exit(1)
```

Comment:
should be sys.exit which is apparently different https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python

Reply:
Done